pkg/adaptation/wasm-disabled.go

```diff
@@ -0,0 +1,27 @@
+//go:build nri_nowasm
```

Comment:
```suggestion
//go:build no_wasm
```

For consistency with `no_btrfs`, `no_systemd`, etc.

https://github.com/containerd/containerd/blob/main/BUILDING.md#build-containerd

Reply:
Or maybe

```suggestion
//go:build no_wazero
```

Reply:
@AkihiroSuda And if we end up flipping the logic around, to have WASM plugin support disabled by default and enabled by a tag, what would be a good build tag name ? enable_wasm_plugins ?

Reply:
For compatibility reason, I don't think we can flip the logic around

Reply:
Okay, reverted build tag to be `nri_no_wasm`.